cmd/manager/main.go

```diff
@@ -674,6 +684,53 @@ func chooseAndValidateIPFamily(ipFamilyStr string, ipFamilyDefault corev1.IPFami
 	}
 }
 
+// determineSetDefaultSecurityContext determines what settings we need to use for security context by using the following rules:
+// 1. If the setDefaultSecurityContext is explicitly set to either true, or false, use this value.
+// 2. use openshift detection pulled from kubevirt
```

Comment:
```suggestion
// 2. use OpenShift detection inspired by kubevirt
```
We have made changes to the original detection code. Also I think the attribution should go on the detection function. 

Maybe we can motivate a little bit more why we don't set the `securityContext` if it is OpenShift so that it becomes clearer to future readers why we implemented this behaviour.

Reply:
👍